### DISCUSSION TITLE
[Question]: Will Frigate have problems if I restore from a partial backup that doesn't include the media folder?

### DISCUSSION THREAD

**Discussion by soonerfan237** [blakeblackshear/frigate#10620] [Ask A Question] [answered]
### What is your question:

I'd like to exclude the media folder from my regular backups to save space. However, I'm unsure what will happen if my Home Assistant server ever dies and I need to restore from this partial backup. Will Frigate face any issues if the media folder isn't there? I'm guessing there's an internal database of events/recordings pointing to folder locations and there may be issues if it suddenly can't find those files. If so, how would I resolve it?

Tangentially related question: does Frigate work well if I move my media folder to a NAS? Any configuration changes or anything else to consider if I go with this approach?

**Comment by NickM-27** [comment, marked as answer]
It will not be able to show existing media for events in the UI, but it won't stop functioning. 

Storing on a NAS won't cause issues

**Reply by soonerfan237** [reply]
Thanks!
